pyproject.toml

```diff
@@ -45,9 +45,9 @@ sphinx-data-viewer = "^0.1.1"  # needservice debug output
 pre-commit = "^2"
 # test dependencies
 lxml = "^4.6.5"
+pytest = "^6.2.5"
```

Comment:
Can you set `>6.2.5`please, otherwise e.g. `6.3.1`is not supported and I don't think that upcoming releases will make trouble.